### PR TITLE
Create input-error component

### PR DESCRIPTION
**Pull request by sebastien-comeau**
As per title.

### CODE REVIEW

File: frontend/app/components/input-error.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import type { ComponentProps, ReactNode } from 'react';
+
+import clsx from 'clsx';
+
+export interface InputErrorProps extends ComponentProps<'span'> {
+  children: ReactNode;
+  id: string;
+}
+
+export const InputError = (props: InputErrorProps) => {
```
Comment (gregory-j-baker) [inline frontend/app/components/input-error.tsx:10]:
```suggestion
export function InputError(props: InputErrorProps) { .. }
```

🏃🏻

File: frontend/app/components/input-label.tsx [hunk 2]
```diff
@@ -1,33 +1,26 @@
...
-export interface InputLabelProps extends Omit<React.ComponentProps<'label'>, ''> {
+export interface InputLabelProps extends ComponentProps<'label'> {
   children: ReactNode;
-  errorId: string;
-  errorMessage?: string;
   id: string;
   required?: boolean;
 }
 
 export const InputLabel = (props: InputLabelProps) => {
```
Comment (gregory-j-baker) [inline frontend/app/components/input-label.tsx:12]:
```suggestion
export function InputLabel(props: InputLabelProps) {
```

**Review by gregory-j-baker — CHANGES_REQUESTED** [review]

File: frontend/app/components/input-error.tsx (hunk 1 above)
Comment (sebastien-comeau) [reply frontend/app/components/input-error.tsx]: Fixed

File: frontend/app/components/input-label.tsx (hunk 2 above)
Comment (sebastien-comeau) [reply frontend/app/components/input-label.tsx]: Fixed

**Review by gregory-j-baker — APPROVED** [review]

**Pull request merged**